repos/system_upgrade/common/models/grubconfigerror.py

```diff
@@ -1,8 +1,9 @@
-from leapp.models import Model, fields
+from leapp.models import fields, Model
 from leapp.topics import SystemFactsTopic
 
 
 class GrubConfigError(Model):
     topic = SystemFactsTopic
 
     error_detected = fields.Boolean(default=False)
+    error_description = fields.StringEnum(['GRUB_CMDLINE_LINUX syntax', 'missing LF'])
```

Comment:
It is not description per say. I would expect description to provide more information describing the actual error and why it is an error. Using StringEnum was a correct choice :+1:, and it also hints that the field is trying to categorize the errors (e.g. tells what kind/type of error it is) rather than describe what the error is.
```suggestion
    error_type = fields.StringEnum(['GRUB_CMDLINE_LINUX syntax', 'missing LF'])
```